### PR TITLE
[BlockLeaves.java] Updated Decay Range

### PR DESCRIPTION
**Pull request by AlexCodesGames**
Updated decay range of leave blocks from 4 to 6 and added variables to allow easier modification of the decay range in the future.

Currently, 'persistent' and 'distance' data has not been changed.

### CODE REVIEW

**Comment by mastercoms** [comment]
Thanks, I'll review this soon. Are you planning to implement the other changes in a new PR?

**Comment by AlexCodesGames** [comment]
Yep! I'm still pretty new and with school back on I'm going to be taking things a little slower than usual to make sure I don't muck anything up too bad. :)

File: src/main/java/net/glowstone/block/blocktype/BlockLeaves.java [hunk 1]
```diff
@@ -19,8 +19,12 @@
...
 
 public class BlockLeaves extends BlockType {
-
-    private final byte[] blockMap = new byte[11 * 11 * 11];
+    
+    private static final int LEAVE_BLOCK_DECAY_RANGE = 6;
+    private static final int LEAVE_BLOCK_MAP_RANGE = (LEAVE_BLOCK_DECAY_RANGE * 2) + 1;
+    
+    //map of neighbouring blocks within decay range, center is current leave block, includes blocks bordering max range
+    private final byte[] blockMap = new byte[Math.ppw(LEAVE_BLOCK_MAP_RANGE + 2, 3)];
```
Comment (mastercoms) [inline src/main/java/net/glowstone/block/blocktype/BlockLeaves.java]: `Math.pow`

**Review by mastercoms — CHANGES_REQUESTED** [review]
Besides that typo, the change looks fine, though I am concerned about the possibility of collisions in the blockMap.

**Review by mastercoms — APPROVED** [review]
Thank you for your contribution to Glowstone!

**Pull request merged**